lib/date_parser.rb

```diff
@@ -1,31 +1,27 @@
 # frozen_string_literal: true
 
 class DateParser
+  attr_reader :raw_date, :issue_prefix, :issues
+
   def initialize(date:, issue_prefix:)
     @raw_date = date.to_h
     @issue_prefix = issue_prefix
-  end
-
-  def issues
-    Requirements::CheckerIssues.new(issue_items)
+    @issues = Requirements::CheckerIssues.new
   end
 
   def parse
-    @issue_items = []
-
+    @issues = Requirements::CheckerIssues.new
     check_date_is_valid
-    issue_items.any? ? return : parsed_date
+    issues.any? ? return : parsed_date
```

Comment:
I'm aware you didn't write most of this line but could this just be changed to - `parsed_date if issues.empty?`?

Reply:
Good point. The return is more explicit, but your suggestion is more idiomatic, so I've changed it.

Reply:
👍